lms/static/js/edxnotes/views/tabs/tags.js

```diff
@@ -0,0 +1,117 @@
+;(function (define, undefined) {
+'use strict';
+define([
+    'gettext', 'jquery', 'underscore', 'js/edxnotes/views/note_group', 'js/edxnotes/views/tab_panel',
+    'js/edxnotes/views/tab_view'
+], function (gettext, $, _, NoteGroupView, TabPanelView, TabView) {
+
+    var TagsView = TabView.extend({
+        scrollToTag: function(tagName) {
+            var titleElement;
+            if (!this.tabModel.isActive()) {
+                this.tabModel.activate();
+            }
+            titleElement = this.$el.find('.tags-title').filter(
+                // Since tags can only be single words, we are safe just looking for the title
```

Comment:
Since you're going to have to manage a dict anyway, can you switch the code to find the element by unique id, or add a unique anchor? I never liked the idea of doing text matches to find the correct element.